client/components/faq/docs/example.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { FAQ, FAQItem } from 'components/faq';
+
+export default React.createClass( {
+	displayName: 'FAQ',
+
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/design/faq">FAQ</a>
+				</h2>
+				<FAQ>
```

Comment:
Right... one row is not enough to show how it aligns. In this design the next row aligns horizontally, leaving varying gaps between items in a single column. But that's not the only possibility... you could also have each item appear with a fixed space between items in a given column, but they would not align as rows then. Unless we show multiple rows, you wouldn't know that.